credentials/apps/badges/models.py

```diff
@@ -536,10 +536,11 @@ def groups(self):
         Returns gorups and their statuses (fulfilled or not).
         """
 
-        return {
-            group: BadgeRequirement.is_group_fulfilled(group=group, template=self.template, username=self.username)
-            for group in self.template.groups
-        }
+        if self.template and self.template.origin == "accredible":
+            return {
+                group: BadgeRequirement.is_group_fulfilled(group=group, template=self.template, username=self.username)
+                for group in self.template.groups
```

Comment:
Can't we do it like this instead of adding a check?
`for group in getattr(self.template, "groups")`